terraform/gcp/projects/pilot-hubs.tfvars

```diff
@@ -36,6 +36,14 @@ notebook_nodes = {
       type: "",
       count: 0
     }
+  },
+  "ohw": {
+    min: 0,
+    max: 100,
+    machine_type: "n1-highmem-4",
+    labels: {
+      "2i2c.org/community": "ohw"
+    },
```

Comment:
We _might_ need the same gpu key here as we have for the dask_nodes? Not sure though - I hope terraform plan would fail informatively here (or check the variables.tf file)